packages/remediations/src/newSteps/selectPlaybook.scss

```diff
@@ -6,5 +6,11 @@
 }
 
 .ins-c-remediations-select-playbook-description { 
-    margin-left: var(--pf-global--spacer--md);;
-}
\ No newline at end of file
+    margin-left: var(--pf-global--spacer--md);
```

Comment:
you can use PF class names for these: 'pf-u-ml-md'. You can interpret the `ml` like **m** is margin or padding (m/p) and the **l** os optional position of the spacing. None means all four sides and then you have t, b, r, l for a specific side.

So there is no need for custom classes just for spacing.

Reply:
Thanks!